controls/SV-230274.rb

```diff
@@ -61,17 +61,18 @@
     describe 'Control not applicable within a container' do
       skip 'Control not applicable within a container'
     end
-  else
-    describe file('/etc/sssd/sssd.conf') do
-      it { should exist }
+  elsif file(input('sssd_conf_path')).exist?
+    describe parse_config_file(input('sssd_conf_path')) do
+      its('sssd') { should include('certificate_verification' => input('sssd_certificate_verification')) }
```

Comment:
I am not sure if this works with the updated structure you have, but thinking more like this: 
```suggestion
its('sssd.certificate_verification') { should match /ocsp_dgst(\s+)?=(\s+)?sha1/ }
```